edward/inferences/variational_inference.py

```diff
@@ -123,6 +123,10 @@ def update(self, feed_dict=None):
 
     sess = get_session()
     _, t, loss = sess.run([self.train, self.increment_t, self.loss], feed_dict)
+    if self.logging and self.n_print != 0:
+      if t == 1 or t % self.n_print == 0:
+          summary = sess.run(self.summarize, feed_dict)
```

Comment:
not much. i recommend including the `_, t, loss = ...` stuff as part of that if/else statement then.